internal/pkg/service/buffer/worker/upload/upload_test.go

```diff
@@ -87,14 +92,14 @@ func TestSliceUploadTask(t *testing.T) {
 	assert.NoError(t, err)
 
 	// Switch slices to the closing state
-	clk.Add(time.Minute)
+	clk.Add(10 * time.Second)
```

Comment:
Uz sa resync interval predava spravne, takze to tu musim znizit, aby nebolo v logoch `reset`.